sdcm/nemesis.py

```diff
@@ -1183,7 +1183,11 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, host_id=None,
             count=1, dc_idx=self.target_node.dc_idx, enable_auto_bootstrap=True, rack=rack)[0]
         self.monitoring_set.reconfigure_scylla_monitoring()
         self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
-        if new_node.is_replacement_by_host_id_supported:
+
+        # since we need this logic before starting a node, and in use_preinstalled_scylla case
```

Comment:
it has a typo... `in `use_preinstalled_scylla: false` case

Reply:
fixed